### ISSUE TITLE
Too Many Labels Makes Save Button Inaccessible

### ISSUE THREAD

**Issue by NickBardjis** [sgoldenlab/simba#136]
Hi.  I'm trying out SimBA to see if it can help with my team's current project.  The animals we want to track are fairly complex with 31 body parts labeled.  This, of course, isn't one of the pre-defined options for the pose config, so I'm having to create a new one.  The problem is that with so many body parts, the Save Pose Config button gets pushed off screen even with the window at max size, and with no way to scroll down it becomes inaccessible.

I've tried hitting the tab button until it's (I think, I can't see to know for sure) selected and hitting enter, but this doesn't work either (either that or I need to fill in the Image Path section, but I haven't found anything telling me what that is).

**Comment by sgoldenlab** [comment]
Thanks for reporting @NickBardjis - yes, we should make the window scrollable, I will let you know when done

**Comment by NickBardjis** [comment]
Thank you.

By the way, I've noticed the instructions specifically talk about labeling rodents.  Can the program be used for other animals?

**Comment by sgoldenlab** [comment]
Yes I've worked directly with users with birds, moths, fish.. rodents is just what we do in the lab

**Comment by inoejj** [comment]
Please download the latest development wheel and there should be a scrollbar

`pip install simba-uw-tf-dev==0.87.2`

**Comment by NickBardjis** [comment]
Thank you again.  One more question: I found the info I was looking for about the image path, and it says all body parts should be clearly visible in the image, but I'm not sure that's anatomically possible, as we have body parts marked on the front and back and the left and right sides.  I think we might be able to get most of them if things line up perfectly, but not all of them.  How big of an issue is that going to be?

**Comment by inoejj** [comment]
It wont be an issue

**Comment by NickBardjis** [comment]
Okay, thanks again!

**Comment by sgoldenlab** [comment]
@NickBardjis - just to check, are you recording your videos from above or the side of the animal?

**Comment by NickBardjis** [comment]
The side, typically.

**Comment by NickBardjis** [comment]
Oh, also I found a similar issue to my original problem with the Label Behavior window.  Unlike my last problem, it's still usable, but a portion of the UI (specifically the part with all the keyboard shortcuts listed) is pushed halfway off the screen (I assume because my video is too wide).  Again, it's still usable, so not a bid deal, but I figured I should mention it.

By the way, I have a few more questions (and will likely have more as I continue to explore the software).  Is there a better place for me to ask them, since most of them aren't really issues with the software?
